#!/usr/bin/python3

def raise_exception_msg(message=""):
    """RaiseNameErroxception witha message."""
    raise NameError(message)
